port: /dev/ttyACM1
baud: 57600
# Actuator automatically follows the brake (1: true, 0: false)
use_brake_follower: 0
# Baud rate to communicate with Cool Muscle actuator
cool_muscle_baudrate: 38400

# Signal overlap duration when switching gears
signal_overlap_ms: 15

# Parameters for the linear line controlling the voltage for the accelerator
#
# The following equation is used to determine the output voltage:
#   out_voltage = voltage_a_slope * percentage + voltage_a_intercept
# where percentage is [0, 100]. The out_voltage is written twice onto the SPI
# wire, which is connected to the ECU.
ecu_voltage_a: [25.54,  746.00]
ecu_voltage_b: [26.00, 1495.00]
# Rate (hz) to publish the status such as brake percentage, potentiometer
# Set to 0 to disable
# Note: If too fast, may overflow the serial port.
status_rate: 0.0
